pkg/utils/types.go

```diff
@@ -95,40 +97,67 @@ const (
 
 // GenerateClientSetFromKubeConfig will generation both ClientSets (k8s, and Litmus)
 func (clientSets *ClientSets) GenerateClientSetFromKubeConfig() error {
-	config, err := getKubeConfig()
+	configExperiment, configLitmus, err := getKubeConfig()
 	if err != nil {
 		return err
 	}
-	k8sClientSet, err := k8s.GenerateK8sClientSet(config)
+
+	k8sClientSetLitmus, err := k8s.GenerateK8sClientSet(configLitmus)
 	if err != nil {
 		return err
 	}
-	litmusClientSet, err := litmus.GenerateLitmusClientSet(config)
+	litmusClientSetLitmus, err := litmus.GenerateLitmusClientSet(configLitmus)
+	if err != nil {
+		return err
+	}
+
+	k8sClientSetExperiment, err := k8s.GenerateK8sClientSet(configExperiment)
 	if err != nil {
 		return err
 	}
-	clientSets.KubeClient = k8sClientSet
-	clientSets.LitmusClient = litmusClientSet
+	litmusClientSetExperiment, err := litmus.GenerateLitmusClientSet(configExperiment)
+	if err != nil {
+		return err
+	}
+
+	clientSets.KubeClient = k8sClientSetLitmus
+	clientSets.LitmusClient = litmusClientSetLitmus
+	clientSets.KubeClientExperiment = k8sClientSetExperiment
+	clientSets.LitmusClientExperiment = litmusClientSetExperiment
 
 	return nil
 }
 
 // Generate
 
 // getKubeConfig setup the config for access cluster resource
-func getKubeConfig() (*rest.Config, error) {
+func getKubeConfig() (*rest.Config, *rest.Config, error) {
 	kubeconfig := flag.String("kubeconfig", "", "absolute path to the kubeconfig file")
+	litmuskubeconfig := flag.String("litmuskubeconfig", "", "absolute path to the kubeconfig file")
```

Comment:
I suppose we can also distinguish the usage/help message to specify: 

 litmuskubeconfig - "absolute path to the kubeconfig file of target cluster where experiment is launched?"

Reply:
Will do.

Reply:
ref: https://github.com/litmuschaos/chaos-runner/pull/105#discussion_r510660184